docs/history/changelog.md

```diff
@@ -210,6 +210,7 @@ Per the [normative and non-normative content and changes policy](../governance/n
   * [#1490](https://github.com/open-contracting/standard/pull/1490) `SimpleIdentifier`
   * [#1519](https://github.com/open-contracting/standard/pull/1519) `Value.amountNet` and `Value.amountGross`
   * [#1490](https://github.com/open-contracting/standard/pull/1490) `contracts.identifiers`
+  * [#1679](https://github.com/open-contracting/standard/pull/1679) `Organization.details.classifications` merged from Organization classifications extension
```

Comment:
The changelog doesn't mention the other extensions from which fields were merged (submission terms and pagination).

```suggestion
  * [#1679](https://github.com/open-contracting/standard/pull/1679) `Organization.details.classifications`
```